base/system/winlogon/sounds.c

```diff
@@ -0,0 +1,357 @@
+
```

Comment:
```suggestion
/*
 * PROJECT:     ReactOS Winlogon
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     System sound notifications
 * COPYRIGHT:   Copyright 2011 Rafal Harabien <rafalh@reactos.org>
 *              Copyright 2013 Eric Kohl <eric.kohl@reactos.org>
 *              Copyright 2016 Hermès Bélusca-Maïto <hermes.belusca-maito@reactos.org>
 *              Copyright 2021 Thamatip Chitpong <tangaming123456@outlook.com>
 */

```
Based on an existing header from `sas.c` (where portions of code were moved), git blame metadata, and https://reactos.org/wiki/Coding_Style

Reply:
OK.